content/en/docs/setup/production-environment/tools/kubeadm/kubelet-integration.md

```diff
@@ -157,15 +157,19 @@ has finished performing the TLS Bootstrap.
 
 ##  The kubelet drop-in file for systemd
 
-The configuration file installed by the kubeadm DEB or RPM package is written to
-`/etc/systemd/system/kubelet.service.d/10-kubeadm.conf` and is used by systemd.
+`kubeadm` ships with configuration for how the `kubelet` should
```

Comment:
How about: “for how systemd should run kubelet”.

Reply:
Tx; done. PS: Pro Tip, use GitHub's cool [suggest a specific change](https://help.github.com/en/articles/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request) (Pt. 6) feature!